### PR TITLE
SEP-38: fixes and improvements

### PR DESCRIPTION
**Pull request by marcelosalloum**
### What

Make a few updates to SEP-38 for improved clarity and usability:
* Fix example response from the `GET /quote` request.
* Add parameters `sell_asset` and `buy_asset` to the responses from `GET /quote/{id}` because without these fields the request won't provide much valuable information
* Update the response from `POST /quote` to be consistent with the changes added to `GET /quote`
* Make it clear that fees are included in the prices from `GET /prices` and `GET /price`
* Add a statement about the indicative price being an estimate, so the actual price will be calculated at conversion time.
* Fix inconsistency between `delivery_methods` field description and examples.

### CODE REVIEW

File: ecosystem/sep-0038.md [hunk 1]
```diff
@@ -365,66 +375,25 @@ Name | Type | Description
 `id` | string | The `id` specified in the request.
 `expires_at` | [UTC ISO 8601](https://www.iso.org/iso-8601-date-and-time-format.html) string | The date and time by which the anchor must receive funds from the client.
 `price` | string | The price offered by the anchor for one unit of `buy_asset` in terms of `sell_asset`. In traditional finance, `buy_asset` would be referred to as the base asset and `sell_asset` as the counter asset.
-`buy_amount` | string | The amount of `buy_asset` to be exchanged for `sell_asset`. `price * buy_amount = sell_amount` must be true up to the number of decimals required for `buy_asset`.
+`sell_asset` | string | The asset the client would like to sell. Ex. `USDC:G...`, `iso4217:ARS`
 `sell_amount` | string | The amount of `sell_asset` to be exchanged for `buy_asset`.
+`buy_asset` | string | The asset the client would like to exchange for `sell_asset`.
+`buy_amount` | string | The amount of `buy_asset` to be exchanged for `sell_asset`. `price * buy_amount = sell_amount` must be true up to the number of decimals required for `buy_asset`.
 
-##### Examples
```
Comment (JakeUrban) [inline ecosystem/sep-0038.md:371]: Can we keep the examples deleted examples but just move it to the `GET /price` endpoint?

**Review by JakeUrban — APPROVED** [review]
One request to keep but move the current examples, otherwise LGTM

File: ecosystem/sep-0038.md [hunk 2]
```diff
@@ -7,14 +7,14 @@ Author: Jake Urban <@jakeurban> and Leigh McCulloch <@leighmcculloch>
 Track: Standard
 Status: Draft
 Created: 2021-04-09
-Updated: 2021-05-03
+Updated: 2021-06-28
 Discussion: https://github.com/stellar/stellar-protocol/issues/901
-Version 1.1.2
+Version 1.1.3
```
Comment (leighmcculloch) [inline ecosystem/sep-0038.md:12]: It looks like this change adds fields to the responses as new functionality, so this version should be a minor version. i.e. `1.2.0`

File: ecosystem/sep-0038.md [hunk 3]
```diff
@@ -172,6 +172,10 @@ Name | Type | Description
 
 This endpoint can be used to fetch the [indicative](https://www.investopedia.com/terms/i/indicativequote.asp) prices of available off-chain assets in exchange for a Stellar asset and vice versa.
 
+These prices are just indicatives provided at a given time, the actual price will be calculated according with the market price at conversion time, after the Anchor receives the funds from a User.
+
+It is expected that the prices returned **include a margin for the provider as a service fee**, and this margin will vary depending on the directional flow of funds.
```
Comment (leighmcculloch) [inline ecosystem/sep-0038.md:177]: Can you remove the emphasis (bold)? SEPs contain a lot of important information and to bold one piece of information over the other implies there's areas of text that can be ignore.

File: ecosystem/sep-0038.md [hunk 4]
```diff
@@ -233,6 +237,10 @@ This endpoint can be used to fetch the indicative price for a given asset pair.
 
 The client must provide either `sell_amount` or `buy_amount`, but not both.
 
+These prices are just indicatives provided at a given time, the actual price will be calculated according with the market price at conversion time, after the Anchor receives the funds from a User.
```
Comment (leighmcculloch) [inline ecosystem/sep-0038.md:240]:
```suggestion
These prices are indicative. The actual price will be calculated at conversion time, once the Anchor receives the funds from a User.
```

File: ecosystem/sep-0038.md [hunk 5]
```diff
@@ -233,6 +237,10 @@ This endpoint can be used to fetch the indicative price for a given asset pair.
...
+
+It is expected that the price returned **includes a margin for the provider as a service fee**, and this margin will vary depending on the directional flow of funds.
```
Comment (leighmcculloch) [inline ecosystem/sep-0038.md:242]: Same comment regarding emphasis.

File: ecosystem/sep-0038.md [hunk 6]
```diff
@@ -317,7 +325,7 @@ Servers may deny access to the API if misuse is detected.
 
 #### Transaction Fees
 
-It should be expected that the provided quote includes a margin for the provider as a service fee, and this margin will vary depending on the directional flow of funds.
+It is expected for the quote returned to **include a margin for the provider as a service fee**, and this margin will vary depending on the directional flow of funds and the delivery method.
```
Comment (leighmcculloch) [inline ecosystem/sep-0038.md:328]:
```suggestion
The quote returned includes any margin that the provider may keep as a service fee, and this margin may vary depending on the directional flow of funds, the delivery method, amount, or other factors.
```

**Review by leighmcculloch — COMMENTED** [review]
Couple comments, and suggested recordings, otherwise looks great.

File: ecosystem/sep-0038.md (hunk 1 above)
Comment (marcelosalloum) [reply ecosystem/sep-0038.md:371]: There are already plenty of examples for the `GET /price` endpoint [here](https://github.com/stellar/stellar-protocol/blob/master/ecosystem/sep-0038.md#examples-1).

The examples I removed were using `GET /price` requests but showing responses with fields from `GET /quote`, so they wouldn't fit any of those endpoints.

File: ecosystem/sep-0038.md (hunk 2 above)
Comment (marcelosalloum) [reply ecosystem/sep-0038.md]: https://github.com/stellar/stellar-protocol/pull/987/commits/a727baf7a92e2e581a3fbd9c7295759b3a741605

File: ecosystem/sep-0038.md (hunk 3 above)
Comment (marcelosalloum) [reply ecosystem/sep-0038.md]: https://github.com/stellar/stellar-protocol/pull/987/commits/a727baf7a92e2e581a3fbd9c7295759b3a741605

File: ecosystem/sep-0038.md (hunk 4 above)
Comment (marcelosalloum) [reply ecosystem/sep-0038.md]: https://github.com/stellar/stellar-protocol/pull/987/commits/a727baf7a92e2e581a3fbd9c7295759b3a741605

File: ecosystem/sep-0038.md (hunk 5 above)
Comment (marcelosalloum) [reply ecosystem/sep-0038.md]: https://github.com/stellar/stellar-protocol/pull/987/commits/a727baf7a92e2e581a3fbd9c7295759b3a741605

File: ecosystem/sep-0038.md (hunk 6 above)
Comment (marcelosalloum) [reply ecosystem/sep-0038.md]: https://github.com/stellar/stellar-protocol/pull/987/commits/972c678112de0a7ac05ce7d43ba9a9b220c55511

File: ecosystem/sep-0038.md [hunk 7]
```diff
@@ -152,15 +152,15 @@ Name | Type | Description
       "delivery_methods": [
         {
           "name": "cash",
-          "description": "Pick up cash BRL at one of our payout locations."
+          "description": "Deposit cash BRL to one of our payout locations."
```
Comment (leighmcculloch) [inline ecosystem/sep-0038.md:155]: Payout = withdraw, so is this actually a deposit or is this a withdrawal? Payout might be the wrong term.

**Review by leighmcculloch — COMMENTED** [review]
One more question.

**Review by JakeUrban — DISMISSED** [review]
LGTM pending the "payout" word change.

File: ecosystem/sep-0038.md (hunk 7 above)
Comment (marcelosalloum) [reply ecosystem/sep-0038.md]: How does this sound? https://github.com/stellar/stellar-protocol/pull/987/commits/023316ccd06f76af962074c941e0c7db148c1d17

**Pull request merged**